### PR TITLE
Adds new names to the names generator

### PR DESCRIPTION
**Pull request by Projjol-zz**
Added in John Nash and Michael Burry

Signed-off-by: Projjol Banerji <probaner23@gmail.com>

### CODE REVIEW

**Review by boaz0 — COMMENTED** [review]
SGTM but can you add a female name too?

File: pkg/namesgenerator/names-generator.go [hunk 1]
```diff
@@ -431,6 +434,9 @@ var (
 		// Ian Murdock - founder of the Debian project - https://en.wikipedia.org/wiki/Ian_Murdock
 		"murdock",
 
+		//John Nash - father of Game Theory
```
Comment (thaJeztah) [inline pkg/namesgenerator/names-generator.go]: Can you;

- add a link to his wikipedia page; https://en.wikipedia.org/wiki/John_Forbes_Nash_Jr.
- nit: add a space before `John`
- perhaps use his full name, and mention that he was a  mathematician

File: pkg/namesgenerator/names-generator.go [hunk 2]
```diff
@@ -186,6 +186,9 @@ var (
 		// Emmett Brown invented time travel. https://en.wikipedia.org/wiki/Emmett_Brown (thanks Brian Goff)
 		"brown",
 
+		// Michael J Burry - one of the first people to predict the economic fall of 2008
+		"burry",
```
Comment (thaJeztah) [inline pkg/namesgenerator/names-generator.go]: Bit in doubt if this one fits in with the general theme.

Given that we'd also like [Women and Men to be equally represented](https://github.com/moby/moby/blob/master/pkg/namesgenerator/names-generator.go#L106); perhaps you can add a different person instead?

**Review by thaJeztah — CHANGES_REQUESTED** [review]

File: pkg/namesgenerator/names-generator.go (hunk 2 above)
Comment (Projjol) [reply pkg/namesgenerator/names-generator.go]: Sure thing!

**Comment by AkihiroSuda** [comment]
ping @Projjol

**Comment by AkihiroSuda** [comment]
ping @Projjol

**Comment by thaJeztah** [comment]
Let me close this one, because there are some review comments open still

**Pull request closed without merging**